### PR TITLE
Update scroll-link for report page

### PR DESCRIPTION
**Pull request by mk-simsoft**
Scroll-link was not work properly for Cancelled Reports. Updated scroll-link component offset.

Closes [AB#721](https://dev.azure.com/ncia-anet/2aa083a5-af3d-44e1-8c7b-6e9e6b124d91/_workitems/edit/721)

#### User changes
- none

#### Super User changes
- none

#### Admin changes
- none

#### System admin changes
- [ ] anet.yml or anet-dictionary.yml needs change
- [ ] db needs migration
- [ ] documentation has changed
- [ ] graphql schema has changed

### Checklist
  - [x] Described the user behavior in PR body
  - [x] Referenced/updated all related issues
  - [x] commits follow a `repo#issue: Title` title format and [these 7 rules](https://chris.beams.io/posts/git-commit/)
  - [x] commits have a [clean history](https://epage.github.io/dev/commits/), otherwise PR may be squash-merged
  - [ ] Added and/or updated unit tests
  - [ ] Added and/or updated e2e tests
  - [ ] Added and/or updated data migrations
  - [ ] Updated documentation
  - [ ] Resolved all build errors and warnings
  - [ ] Opened debt issues for anything not resolved here

### CODE REVIEW

File: client/src/components/Fieldset.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ const Fieldset = ({
       id={id}
       name={id}
       className="scroll-anchor-container"
-      style={style}
+      style={{ style, marginBottom: title === 'Cancelled Reports' ? '300px' : '' }}
```
Comment (gjvoosten) [inline client/src/components/Fieldset.js:36]: You should probably use `...style` here?

File: client/src/components/Nav.js [hunk 2]
```diff
@@ -42,7 +42,7 @@ export const AnchorNavItem = ({ to, disabled, children }) => {
         utils.pushHash(to)
       }}
       // TODO: fix the need for offset
-      offset={-topbarOffset}
+      offset={-250}
```
Comment (gjvoosten) [inline client/src/components/Nav.js:45]: Why did you change this?

Comment (mk-simsoft) [reply client/src/components/Nav.js:45]: Cause, Cancelled Reports field not come this position and -topbarOffset has no any value or may have a value but not work for Cancelled Reports. So I should change and find the specific value for menu and menu trigger work properly.

Comment (gjvoosten) [reply client/src/components/Nav.js:45]: The height of the TopBar is determined dynamically and the topbarOffset is updated through the context provider in ResponsiveLayout. You're making it a fixed value here, which is incorrect; the offset depends on the height of the TopBar, which is dynamic (can depend on window size, whether there's a general banner text being shown or not, etc.).

File: client/src/components/Fieldset.js (hunk 1 above)
Comment (gjvoosten) [reply client/src/components/Fieldset.js]: Also, the bottom margin for any fieldset is 4rem, but `index.css` makes an exception for (a.o.) `#cancelled-reports fieldset` because we don't want to waste that space at the bottom of a page. So in short, this style override is probably not the way you should do it, and it also doesn't actually solve the issue.
